src/responsiveStyles/responsiveContainer.tsx

```diff
@@ -0,0 +1,26 @@
+import { css } from 'styled-components';
+const ScreenWidth = {
+    tabletMini: { screen: 544, maxWith: 500 },
+    tablet: { screen: 768, maxWith: 720 },
+    laptop: { screen: 992, maxWith: 940 },
+    desktop: { screen: 1200 ,maxWith: 1140 },
+}
+
+const containerPadding = '15px';
+
+export const containerStyle = css`
+    ${Object.entries(ScreenWidth).sort((item1, item2) => {
```

Comment:
This logic will run *every render* since it is defined in a styled component that gets evaluated, I think (you can verify with a simple console.log) - and it has 2 loops... assuming sort is logN, it is O(nlogn) :(

I think in this case it is better to be explicit and specify the 4 media queries instead.

Reply:
Moved to an explicit media queries